quickwit/quickwit-indexing/src/split_store/indexing_split_store.rs

```diff
@@ -244,6 +244,10 @@ impl IndexingSplitStore {
         Ok(())
     }
 
+    pub fn get_merge_policy(&self) -> Arc<dyn MergePolicy> {
```

Comment:
We shouldn't have to go through the split store to get to the merge policy. See my comment above.